autora/cycle/plot_utils.py

```diff
@@ -278,13 +279,13 @@ def plot_results_panel_2d(
     condition_space = _generate_condition_space(cycle, steps=steps)
 
     # Get theory predictions over space
-    d_predictions = _theory_predict(cycle, condition_space)
+    l_predictions = _theory_predict(cycle, condition_space)
 
     # Cycle Indexing
     cycle_idx = list(range(len(cycle.data.theories)))
     if query:
         if isinstance(query, list):
-            cycle_idx = query
+            cycle_idx = [cycle_idx[s] for s in query]
```

Comment:
Ah, I changed this because the labeling of the plot cycles was displaying as "Cycle -1" when using negative indexing. By filtering on the positive ordered indexes, we preserve the numbers for the labeling.

Reply:
Ah, I changed this because the labeling of the plot cycles was displaying as "Cycle -1" when using negative indexing. By filtering on the positive ordered indexes, we preserve the numbers for the labeling.

Reply:
aaaaah, very clever. 💯